python/fds.protobuf.stach.v3/setup.py

```diff
@@ -1,12 +1,12 @@
 from setuptools import setup, find_packages
 
-setup(name='fds.protobuf.stach.v2',
-      version='1.0.2',
+setup(name='fds.protobuf.stach.v3',
+      version='1.0.0',
       description='Stach library in python',
       url='',
       author='analytics-reporting',
       author_email='analytics.api.support@factset.com',
       license='Apache License 2.0',
       install_requires=["protobuf>=3.12.3,<4.0.0"],
```

Comment:
Is this the updated version after the issue of not generating the complete content?